bin/node-template/runtime/src/lib.rs

```diff
@@ -234,6 +234,12 @@ impl pallet_timestamp::Config for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
+	/// This constant can also be written in-place as `ConstU128<500>`.
+	/// However, in this case its needed in another place as well.
+	pub const ExistentialDeposit: u128 = 500;
+}
+
```

Comment:
Maybe we should be doing `pub const ExistentialDeposit: u128 = 500;`  and then using `ConstU128<ExistentialDeposit>` ditching the parameter_types macro so that it's more rusty? 
(assuming rustc allows this)